osquery/tables/system/darwin/homebrew_packages.cpp

```diff
@@ -77,29 +81,35 @@ void packagesFromPrefix(QueryData& results,
     return;
   }
 
-  auto cellarPath = fs::path(prefix) / "Cellar";
-
-  if (!pathExists(cellarPath).ok()) {
-    if (userRequested) {
-      LOG(WARNING) << "Error reading homebrew cellar path "
-                   << cellarPath.native();
+  std::tuple<std::string, fs::path> typesList[2] = {
+      std::make_tuple("formulae", fs::path(prefix) / "Cellar"),
```

Comment:
I suggest we use the singular "formula" to match the singular "cask" -- I think it's also just going to be more familiar for users.

```suggestion
      std::make_tuple("formula", fs::path(prefix) / "Cellar"),
```

Reply:
Good call